catalyst/dl/experiment/config.py

```diff
@@ -11,7 +11,10 @@
 )
 
 from catalyst.data import Augmentor, AugmentorCompose
-from catalyst.dl import Callback, Experiment, utils
+from catalyst.data.sampler import DistributedSamplerWrapper
```

Comment:
could you please make it simpler? 
like`from catalyst.data import Augmentor, AugmentorCompose, DistributedSamplerWrapper `?

Reply:
done